runtime/makelib/targets.mk.win.inc.ftl

```diff
@@ -100,6 +100,9 @@ endif
 
 UMA_RC_OPTIONS+=$(UMA_C_INCLUDES)
 
+# Add runtime include path for resource compiler on Windows.
+UMA_RC_OPTIONS:=$(addprefix $(UMA_C_INCLUDE_PREFIX),$(UMA_PATH_TO_ROOT)include) $(UMA_RC_OPTIONS)
```

Comment:
What specifically is causing the problem? This isn't required for IBM builds that create Windows 32-bit builds. What is different about OpenJ9 builds?

Reply:
What is an example directory where the problem occurs? Can you explain why all the OpenJ9 .rc files compile in an IBM build? Perhaps the module.xml needs to be updated in some OpenJ9 specific directory?

Reply:
Investigation shows the problem (failure to locate openj9/runtime/include) only occurs on Cygwin 32-bit version  as follows:
win.rc(4) : fatal error RC1015: cannot open include file 'j9cfg.h'.
make[3]: *** [../makelib/targets.mk:384: win.res] Error 1
make[2]: *** [makefile:128: win] Error 2
make[1]: *** [OpenJ9.gmk:321: build-j9] Error 2
It gets compiled only if the path to  openj9/runtime/include is specified in $(UMA_C_INCLUDES).

It works good without any RC-related issue on Cygwin 64-bit version. So there is no need to change any .mk file here as we use Cygwin 64-bit version for compilation.

Reply:
The root cause is the include folder wasn't specified the runtime/win/module.xml (32bit) as follows:
<includes>
		<include path="j9include"/>
</includes>

With the update, the build got compiled as expected.